packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -84,6 +84,53 @@ await Promise.all([1, 2, 3].map(async x => x + 1));
 
 ## Options
 
+### `checkThenables`
+
+A ["Thenable"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise#thenables) value is an object which has a `then` method, such as a `Promise`.
+Other Thenables include TypeScript's built-in `PromiseLike` interface and any custom object that happens to have a `.then()`.
+
+The `checkAllThenable` option triggers `no-floating-promises` to also consider all values that happen to be Thenable, not just Promises.
```

Comment:
nit - technically this is very slightly false (since it only considers thenables that can catch a rejected state with a second parameter), but I doubt anyone cares.

Reply:
Good point! I'll fix it just to be safe.